### ISSUE TITLE
Lift stepper never returns to bus after flash on Python3.8 Ubuntu 20.04

### ISSUE THREAD

**Issue by hello-binit** [hello-robot/stretch_factory#30]
After flashing on Ubuntu 20.04, the lift never returns to the bus. This causes the lift's calibration information to never be flashed to the lift's EEPROM.

```
-------- FIRMWARE FLASH hello-motor-lift | Stepper.v0.1.0p1 ------------
Starting programming. This will take about 5s...
Success in firmware compile
Success in firmware flash.
Waiting for device hello-motor-lift to return to bus.
.................Failure for device hello-motor-lift to return to USB bus after upload
```

This happens consistently when flashing from Ubuntu 20.04.

**Comment by aedsinger** [comment]
Confirmed this issue as well. The following WIP branch is trying to find a solution:

https://github.com/hello-robot/stretch_factory/tree/bugfix/udev_rule

**Comment by aedsinger** [comment]
Fixed in 0.2.3. Tested 100 cycles on all boards with no errors (on 20.04)

**Comment by hello-binit** [comment]
Seeing the same issue on Ubuntu 20.04, but with additional output from usb.core this time:

```
-------- FIRMWARE FLASH hello-motor-lift | Stepper.v0.2.5p1 ------------
Resetting hello-motor-lift
Looking for device hello-motor-lift on bus
Waiting for device hello-motor-lift to return to bus.
Starting programming. This will take about 5s…
Success in firmware compile
Success in firmware flash.
ls parse: Failed to match all devices for StretchSerialInfo
{‘hello-motor-lift’: {‘device’: None, ‘info’: None, ‘core’: None}}
usb.core parse: Failed to match all devices for StretchSerialInfo
Waiting for device hello-motor-lift to return to bus.
…Failure for device hello-motor-lift to return to USB bus after upload
```

**Comment by hello-binit** [comment]
This issue is fixed as of Stretch Factory v0.3.9. Use `pip3 install -U hello-robot-stretch-factory` to update to this version on Ubuntu 20.04
